src/stavetempo.ts

```diff
@@ -2,17 +2,21 @@
 // @author: Radosaw Eichler 2012
 
 import { Element } from './element';
+import { Glyphs } from './glyphs';
 import { Metrics } from './metrics';
 import { Stave } from './stave';
 import { StaveModifier, StaveModifierPosition } from './stavemodifier';
 import { Tables } from './tables';
 import { Category } from './typeguard';
 
 export interface StaveTempoOptions {
-  bpm?: number;
+  name?: string;
+  parenthesis?: boolean;
   duration?: string;
   dots?: number;
-  name?: string;
+  bpm?: number;
+  duration2?: string;
+  dots2?: number;
```

Comment:
Can we add a comment somewhere in this file explaining what duration2 and dots2 is for?

You can explain it is for cases where the tempo marker is X = Y where both X and Y are notes (not BPM numbers).

Reply:
done